akka-http-core/src/main/resources/reference.conf

```diff
@@ -125,6 +125,16 @@ akka.http {
     parsing {
       # no overrides by default, see `akka.http.parsing` for default values
     }
+
+    # Enables/disables the logging of HTTP request/response exchange. If enabled
+    # each raw incoming request and outgoing response will be logged using human
+    # readable format.
+    exchange-logging = on
```

Comment:
Can we unify `exchange-logging` and `exchange-logging-bytes`? 

Maybe one setting called `log-unencrypted-network-bytes` that is set to `off` by default?